src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);

```

Comment:
Here, your random instance is created every time the method is called. random - is our dependency, let's initialize dependencies using a constructor or directly create a new instance in the class fields.